src/pyscipopt/scip.pyx

```diff
@@ -3075,11 +3075,13 @@ cdef class Model:
         """
         SCIPbendersSetSubproblemIsConvex(benders._benders, probnumber, isconvex)
 
-    def setupBendersSubproblem(self, probnumber, Benders benders = None, Solution solution = None):
+    def setupBendersSubproblem(self, probnumber, checktype, Benders benders = None, Solution solution = None):
```

Comment:
wanna give a default value to checktype?

Reply:
I thought about it, but didn't know what the default should be because the most common are `LP` and `CHECK`. I think by making a default, it doesn't highlight to the user that a different option will be needed in some circumstances.